doc/releases/release-notes-3.6.rst

```diff
@@ -261,6 +263,29 @@ Drivers and Sensors
   * The Legacy Virtual Register defines have been renamed from ``I3C_DCR_I2C_*``
     to ``I3C_LVR_I2C_*``.
 
+  * Added the ability to specify a start address when searching for a free I3C
+    address to be reserved. This requires a new function argument to
+    :c:func:`i3c_addr_slots_next_free_find`.
+
+  * ``i3c_cdns``:
+
+    * Added support to handle controller abort where target does not emit
+      end of data for register read but continues sending data.
+
+    * Updated the timeout calcultion to be coupled with CPU speed instead of
+      a fixed number of retries.
+
+  * ``i3c_mcux``:
+
+    * Fixed config_get of not returning the configuration.
```

Comment:
sentence is broken? (unless I'm missing something as a non-native speaker, if so I'm sorry!)

Reply:
Clarified it a bit. I was thinking about function returning some values... and it seemed clear to me.